angular-app2/src/styles.css

```diff
@@ -0,0 +1,18 @@
+/* You can add global styles to this file, and also import other style files */
+* {
+    margin: o;
```

Comment:
Trebuie margin 0 in loc de o :) . O data ce schimbi asta se reareanjeaza putin aplicatia.